File: project/src/main/java/iteration2/project/CourseSection.java
package iteration2.project;

import java.util.ArrayList;
import java.util.HashMap;

public class CourseSection {
	private int quota;
	private HashMap<String, ArrayList<String>> schedule;
	
	public CourseSection(int quota) {
		this.quota = quota;
		this.schedule = new HashMap<String, ArrayList<String>>();
	}
	
	public void addToSchedule(String day, String hours) {
		if(schedule.get(day) == null) {
			schedule.put(day, new ArrayList<String>());
		}
		schedule.get(day).add(hours);	
	}

	
	public int getQuota() {
		return quota;
	}

	public void setQuota(int quota) {
		this.quota = quota;
	}

	public HashMap<String, ArrayList<String>> getSchedule() {
		return schedule;
	}
	
}

File: py_iteration/CourseSection.py
class CourseSection:
    def __init__(self, quota):
        self.quota = quota
        self.schedule = {}  # key: day, value: [hours] --> hours: "11-13"

    def add_to_schedule(self, day, hours):
        if day not in self.schedule:
            self.schedule[day] = []
        self.schedule[day].append(hours)